run.py

```diff
@@ -2,15 +2,24 @@
 import team_comps
 
 
-
 user_input = []
 
+
 def get_user_input():
     user_input_string = input("What champions do you have?")
-    team_comps.build_data(user_input_string)
+    user_input_string = user_input_string.split(" ")
+
+    print(user_input_string)
+
+    for i in user_input_string:
+        if '-' in i:
+            team_comps.delete_champ(i)
+        else:
+            team_comps.add_champ(i)
+
     calculate.match(user_input, team_comps.tier_S)
```

Comment:
I feel like we should move this into another function, or at least rename this function. It does more than just get user input.